front/lib/resources/app_resource.ts

```diff
@@ -42,47 +38,128 @@ export class AppResource extends ResourceWithVault<App> {
       vaultId: vault.id,
     });
 
-    return new this(AppResource.model, app.get(), vault, []);
+    return new this(AppResource.model, app.get(), vault);
+  }
+
+  // Fetching.
+
+  // baseFetch filters out private apps if the auth is not a user on the workspace. This will be
```

Comment:
Comment is misplaced will have to update it before merging

front-edge looks good